src/devices/bosch.ts

```diff
@@ -1817,6 +1843,10 @@ const definitions: Definition[] = [
                 e.enum('motor_state', ea.STATE, Object.keys(stateMotor))
                     .withDescription('Shutter motor actual state '),
                 e.binary('child_lock', ea.ALL, 'ON', 'OFF').withDescription('Enable/Disable child lock'),
+                e.numeric('pushButtonHoldTime', ea.ALL).withUnit('s')
```

Comment:
Same as for calibration up and down time, you can also set e.g. 0,1 for 100ms. In my test I set 0,2 for pushButton and 0 for delay and it worked good. Using milliseconds directly I guess does not work, since you can not set 0,1 as true value. 0 works fine and 1 = 100ms on the cluster.

Reply:
I must have made a mistake on my last test, thanks for the catch 😄 Max value on both clusters is 255, i also added .valueStep(0.1)
I tested it writing the value instead of using the slider, this worked fine. 
Another approach would be .valueStep(100) and size everything down to ms, do you think this would be more comfortable? I don't realy care which one to choose.

Reply:
And 0.05 does not work btw.